### ISSUE TITLE
CTRDataSyncFunction  - Won't process if Voicemail is left in 9.1

### ISSUE THREAD

**Issue by Semag** [service-cloud-voice/ServiceCloudVoiceLambdas#21]
Hey all - I'm not sure if this is the right place to report this (I also have a Salesforce Ticket open) but I have been banging my head against the wall for a while trying to figure out why a bunch of our fields on voice calls weren't updating.

It turns out the new version of CTRDataSyncFunction  has a commented line that says this:
```
      const isVM = ctr.Attributes && ctr.Attributes.vm_flag;
      **// if the ctr data sync has the Voicemail flag, no need to process it since the packager lambda will update the voicecall record**
      if (ctr.ContactId && !isVM) {
        const voiceCall = utils.transformCTR(ctr);
```

I moved our contact center back to the old version, but in this project i also noticed you have version 9.0 and not 9.1 in the project?

**Comment by jinalkathiara** [comment]
I added 9.1 version recently. We are not having tags in the github but if you see latest commits, you should be able to see 9.1 version. 
Sorry for the breakage and can you mention your salesforce support case # ? We can guide you and update our support org as well. 

Regarding the issue you are talking about, can you elaborate more ?

**Comment by Semag** [comment]
Thanks for the reply @jinalkathiara  - the 9.1 version of CTRDataSyncFunction that you have in this repository doesn't seem to be the same as the one that auto upgraded in our AWS instance.

Basically, the issue that we had is that after the upgrade on 8/9 in AWS, our VoiceCallRecords were no longer updating with callAttributes for all calls that resulted in a voicemail.  I didn't quite recognize this as the symptom until I found the root cause today.

The reason that our voiceCallRecords are not updating, is that the newest version of handler.js in the CTRDataSyncFunction is literally checking to see if the call was a voicemail, and if so, it skips the update.  I backed the version back down to the previous version and everything started working as it previously did.
